clients/lsp-qml.el

```diff
@@ -0,0 +1,43 @@
+;;; lsp-qml.el --- description -*- lexical-binding: t; -*-
```

Comment:
Can you put a reasonable description? Thanks!

Reply:
Sure 🙂 Added a short version of the one I put in "Commentary"